packages/astro/src/core/build/index.ts

```diff
@@ -89,6 +91,13 @@ class AstroBuilder {
 			command: 'build',
 			logging,
 		});
+
+		// HACK: Since we only inject the endpoint if `experimental.assets` is on and it's possible for an integration to
```

Comment:
Is it worth a “Remove in 3.0” comment or would we pick it up anyway because `experimental.assets` will go away so this would be a type error?

Reply:
In 3.0 it's likely that we'll need an option to disable the endpoint (ex: maybe you have your own, maybe you don't want any `assets` stuff), so we'll still need to check for something in the config still here. So for now I'll leave it like this and update whenever we'll add that option